demos/common/python/models/monodepth.py

```diff
@@ -0,0 +1,37 @@
+"""
+ Copyright (C) 2018-2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import cv2
+
+from .segmentation import SegmentationModel
+
+
+class MonoDepthModel(SegmentationModel):
+    def postprocess(self, outputs, meta):
+        result = outputs[self.out_blob_name].squeeze()
```

Comment:
```suggestion
        result = outputs[self.output_blob_name].squeeze()
```
(Since it is changed in **Model API refactoring** PR)